pkg/referrerstore/oras/oras.go

```diff
@@ -61,6 +60,8 @@ type orasStore struct {
 	localCache *content.OCI
 }
 
+var authProvider authprovider.AuthProvider
```

Comment:
Should this be a member of `orasStore`?

Reply:
good point. I'll add it to `orasStore` instead.